<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonHit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>740</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Взять карту</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/game/images/resources/take_card.png</normaloff>:/game/images/resources/take_card.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStand">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>740</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Пропустить</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/game/images/resources/skip_card.png</normaloff>:/game/images/resources/skip_card.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNewGame">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>740</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> Новая игра</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources.qrc">
      <normaloff>:/game/images/resources/new_game.png</normaloff>:/game/images/resources/new_game.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="playerIcon">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources.qrc">:/game/images/resources/player.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="botIcon">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources.qrc">:/game/images/resources/bot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlayerScore">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>219</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBotScore">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>610</y>
      <width>219</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="widgetPlayerCards"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="widgetBotCards"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="actions">
    <property name="title">
     <string>Действие</string>
    </property>
    <addaction name="actionHit"/>
    <addaction name="actionStand"/>
    <addaction name="actionNewGame"/>
   </widget>
   <widget class="QMenu" name="rules">
    <property name="title">
     <string>Правила</string>
    </property>
    <addaction name="actionRule1"/>
    <addaction name="actionRule2"/>
   </widget>
   <widget class="QMenu" name="music">
    <property name="title">
     <string>Музыка</string>
    </property>
    <addaction name="actionUnmute"/>
    <addaction name="actionMute"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="actions"/>
   <addaction name="rules"/>
   <addaction name="music"/>
  </widget>
  <action name="actionHit">
   <property name="text">
    <string>Взять карту</string>
   </property>
  </action>
  <action name="actionStand">
   <property name="text">
    <string>Пропустить</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>Новая игра</string>
   </property>
  </action>
  <action name="actionRule1">
   <property name="text">
    <string>Против бота</string>
   </property>
  </action>
  <action name="actionRule2">
   <property name="text">
    <string>Против другого игрока</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionUnmute">
   <property name="text">
    <string>Включить звук</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Выключить звук</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
